app/src/main/java/com/infomaniak/drive/ui/fileList/SearchFragment.kt

```diff
@@ -212,7 +212,7 @@ class SearchFragment : FileListFragment() {
     }
 
     private fun configureRecentSearches() = with(binding) {
-        fileListLayout.addView(recentSearchesBinding.root, 1)
+        fileListLayout.addView(recentSearchesBinding.root, 2)
```

Comment:
So that's the fix. This is so error prone actually. Shouldn't we include the layout directly inside fragment_file_list.xml and only show it when needed instead?